stylesheets/scss/_layout.scss

```diff
@@ -2,3 +2,45 @@
 	Layout goes in this file. Susy setup should go here too.
 	Layout classes should be prefixed `.l-`
 */
+
+// Main Configuration options for the Susy Grid
+$gutter-width: 20px;
+$column-count: 12;
+
+$susy: (
+  columns: $column-count,
+  use-custom: (rem:true),
+  global-box-sizing: border-box,
+  gutter-position: inside-static,
+  column-width: 96px,
+  gutters: $gutter-width/96px,
+  // debug: (
+  //   image: show-columns,
+  //   output: overlay,
+  //   toggle: bottom left,
+  // )
+);
+
+%l-container,
+.l-container {
+  //@include container($breakpoint-largest);
+  @include clearfix;
+  max-width: $breakpoint-large;
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: $body-gutter - gutter();
+  padding-right: $body-gutter - gutter();
```

Comment:
If this is used to give some extra space between the body and the container, why not use `bleed()`? Here's something I've done in `.container` to provide that extra space:
```
.container {
	@include container();
	@include bleed(0 gutter());
	...
```